stestr/commands/init.py

```diff
@@ -38,8 +40,11 @@ def init(repo_type='file', repo_url=None):
         for failures.
     :rtype: int
     """
-
-    util.get_repo_initialise(repo_type, repo_url)
+    try:
+        util.get_repo_initialise(repo_type, repo_url)
+    except FileExistsError:
+        stdout.write('The specified repository already exists\n')
+        return 0
```

Comment:
yeah, I was thinking +1 for a failure condition here originally. But I checked some commands behavior.

* git: success (git init && git init; echo $?)
* hg: failure (hg init && hg init; echo $?)
* mkdir: success (mkdir foo && mkdir foo; echo $?)
* tempest: failure (tempest init foo && tempest init foo; echo $?)

And I found that git-init document[1], it says,
```
Running git init in an existing repository is safe. It will not overwrite things that are already there. The primary reason for rerunning git init is to pick up newly added templates (or to move the repository to another place if --separate-git-dir is given).
```
I think this is a user-friendly interface. So, my conclusion is we can return 0 here :) 

[1] https://git-scm.com/docs/git-init